scripts/lib/LinkChecker.ts

```diff
@@ -117,6 +117,52 @@ export class Link {
     );
   }
 
+  /**
+   * Returns a string with a suggested replacement for a broken link
+   */
+  didYouMean(existingFiles: File[], originFile: string): string {
+    const levenshtein = require("fast-levenshtein");
```

Comment:
This should be an ES6 import, like the rest of our imports. `require` is the old way of doing imports.

Reply:
Done!